apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/test/java/gov/cms/bfd/pipeline/rda/grpc/source/FissClaimTransformerTest.java

```diff
@@ -1880,6 +1881,13 @@ public void testAudiTrailBadtCurrDate() {
             RdaFissAuditTrail.Fields.badtCurrDate);
   }
 
+  @Test
+  public void testAudiTrailRdaPosition() {
+    new AuditTrailFieldTester()
+        .verifyIntFieldCopiedCorrectly(
```

Comment:
It might be better to add a new `verifyUIntFieldToShortFieldCopiedCorrectly` (or better name) method to verify the bounds checks are applied correctly.